### ISSUE TITLE
[Feat]: Løsning på søkeknapp i ny header

### ISSUE THREAD

**Issue by Lillebo** [Informasjonsforvaltning/fdk-frontend#122]
### Feature beskrivelse

Ref. https://frontpage-poc.staging.fellesdatakatalog.digdir.no/

Vi trenger å finne ut hvordan denne knappen skal fungere:

<img width="391" alt="Screenshot 2024-06-25 at 11 05 41" src="https://github.com/Informasjonsforvaltning/fdk-frontend/assets/74510/e9db8490-17e6-4094-99d8-9626fa5b3d79">

Utfordringer:
- Knappen slik den er nå kan skape en forventning om at den tilbyr søk for **alt innhold på hele nettsiden** (inkl. dokumentasjon og øvrige sider utover bare treff i datakataloger)
- Vi har ikke søkefunksjonalitet for annet enn datakataloger per nå, og rekker vel ikke å utvikle eller tilrettelegge for dette før 1. oktober uansett?

Det langsiktige målet er å ha ett samlet søk for hele nettsiden, hvor både dokumentasjon og treff i datakataloger kan søkes på samme sted - gjerne også med integrert AI-søk.

[Sverige](https://www.dataportal.se/sv/datasets?p=1&q=data&s=2&t=20&f=&rt=dataset%24esterms_IndependentDataService%24esterms_ServedByDataService&c=false) og [Danmark](https://www.opendata.dk/search/content) har allerede slike løsninger, og vi burde ha ambisjon om å få til noe tilsvarende eller bedre.

### Forslag/løsning

Frem til vi får tid til å utvikle nettsidesøk og integrere dette med både datakataloger og ai, så er vi begrenset til å kun tilby søk i datakatalogene. Vi er også begrenset med hvor mye tid vi har frem til 1. oktober hvor dette skal være i produksjon.

Jeg foreslår derfor en enkel midlertidig løsning:

- Søkeknappen døpes om til "Finn data" og er kun en lenke inn til: https://data.norge.no/search-all
    - Dette vil forhåpentligvis unngå at brukeren får en forventning om at vi tilbyr noe annet søk enn det vi faktisk gjør per nå
- "Gå til avansert søk"-knappen i bunnen av AI-resultatene døpes også om til "Finn data". Disse knappene vil da ha akkurat samme funksjonalitet og burde derfor hete det samme. Forhåpentligvis vil dette betrygge brukeren om at h*n ikke behøver å gå via AI-søket for å finne frem til dette.
- Søkeknappen på AI-søket døpes om til bare "Spør" for å differensiere denne fra de nye inngangene til søkesiden som nå bruker denne merkelappen.

<img width="1587" alt="Screenshot 2024-06-25 at 12 21 52" src="https://github.com/Informasjonsforvaltning/fdk-frontend/assets/74510/b6ca2d81-261f-494c-80e1-02e5801adbdb">
<img width="1434" alt="Screenshot 2024-06-25 at 12 22 29" src="https://github.com/Informasjonsforvaltning/fdk-frontend/assets/74510/c2e41174-9c70-4390-8a84-cc2861c5937d">

**Comment by terjesyl** [comment]
Bra forslag! Syns det virker bra.

**Comment by KurtStian** [comment]
Ser bra ut, men burde knappen nederst i KI-resultat hete «Gå til søket Finn data»? For å tydeliggjøre at dette er et søk? I toppmenyen vises søkeikon så der er det OK.

**Comment by Lillebo** [comment]
Hm, skjønner litt hva du mener @KurtStian 🤔 Hva om vi legger inn søkeikonet her også? Enten i stedenfor, eller i kombinasjon med pila? Liker litt at pila gir et hint om at dette er en lenke som går til en annen side, men det blir kanskje litt mye med begge deler.

<img width="1545" alt="Screenshot 2024-07-02 at 08 46 14" src="https://github.com/Informasjonsforvaltning/fdk-frontend/assets/74510/128773b6-5408-435f-ae23-f35488050ebb">

**Comment by KurtStian** [comment]
@Lillebo 
> Hm, skjønner litt hva du mener @KurtStian 🤔 Hva om vi legger inn søkeikonet her også? Enten i stedenfor, eller i kombinasjon med pila? Liker litt at pila gir et hint om at dette er en lenke som går til en annen side, men det blir kanskje litt mye med begge deler.
> 
Ja, det kan fungere. Jeg tror ikke det blir for mye å ta med pila og det gjør det mer tydelig for bruker at det er en lenke. Men hvis vi gjør det så er det sikkert også andre knappelenker som også må få en slik pil, slik at det blir lik design for disse.

**Comment by Lillebo** [comment]
Ref. det vi snakket om i møtet så har jeg mer lyst til å fjerne pilen fullstendig enn at denne må inn på alle knappene som er lenker. Her er skjermbilder for å sammenligne med/uten:
<img width="1512" alt="Screenshot 2024-07-02 at 15 00 52" src="https://github.com/Informasjonsforvaltning/fdk-frontend/assets/74510/f5cfe9d6-b86f-41bc-b9eb-2a91ddd6a2b1">

Skal vi gå for den til venstre?
